packages/stepper/src/types.ts

```diff
@@ -0,0 +1,63 @@
+import React, { Dispatch } from 'react';
+
+export interface StepperContextValues {
+  isDarkMode: boolean;
+  setIsDarkMode: Dispatch<React.SetStateAction<boolean>>;
+  stepIconClassName: string;
+  stepLabelClassName: string;
+}
+
+// TODO: replace with common type across all components
+export const Mode = {
+  Light: 'light',
+  Dark: 'dark',
+} as const;
+
+export enum StepStates {
```

Comment:
We've had some TS issues with `enum` in the past. That's why we prefer the `const X = {...} as const` pattern

Reply:
Oh yeah the `const` pattern is fine imo. I was more referring to how all this should probably be somewhere globally accessible, not redefined in every component.

Reply:
If this is only used internally then it's fine to keep it as an `enum`, but anything we export from `index` would need to be a `const as const`